File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/com/main/pythondepmanagement_1/graph/Lookup_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from com.main.pythondepmanagement_1.config.ConfigStore import *
from com.main.pythondepmanagement_1.udfs.UDFs import *

def Lookup_1(spark: SparkSession, in0: DataFrame):
    keyColumns = ['''c- short''', '''c  - int''']
    valueColumns = ['''c-string''']
    createLookup("Lookup_1", in0, spark, keyColumns, valueColumns)

File: testprojects/master_scala_main/pipelines/ScalaGCP/code/src/main/scala/graph/everythingSG_1/Lookup_1.scala
package graph.everythingSG_1

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object Lookup_1 {

  def apply(spark: SparkSession, in0: DataFrame): Unit =
    createLookup("Lookup_1",
                 in0,
                 spark,
                 List("customer_id", "first_name"),
                 "last_name",
                 "phone"
    )

}
